test/parallel/test-cluster-disconnect.js

```diff
@@ -1,41 +1,41 @@
 'use strict';
-var common = require('../common');
-var assert = require('assert');
-var cluster = require('cluster');
-var net = require('net');
+const common = require('../common');
+const assert = require('assert');
+const cluster = require('cluster');
+const net = require('net');
 
 if (cluster.isWorker) {
-  net.createServer(function(socket) {
+  net.createServer((socket) => {
     socket.end('echo');
   }).listen(common.PORT, '127.0.0.1');
 
-  net.createServer(function(socket) {
+  net.createServer((socket) => {
     socket.end('echo');
   }).listen(common.PORT + 1, '127.0.0.1');
 
 } else if (cluster.isMaster) {
   var servers = 2;
 
   // test a single TCP server
-  var testConnection = function(port, cb) {
-    var socket = net.connect(port, '127.0.0.1', function() {
+  const testConnection = function(port, cb) {
+    var socket = net.connect(port, '127.0.0.1', () => {
       // buffer result
       var result = '';
-      socket.on('data', function(chunk) { result += chunk; });
+      socket.on('data', common.mustCall((chunk) => { result += chunk; }));
```

Comment:
I'm also not going to stand in the way of this landing over that issue, but I agree with @santigimeno that there's value in accounting for the possibility of multiple `data` events even if it is something that will never actually happen.

People should be able to use our tests as model code. Obviously we can't *always* accomplish that. Sometimes, we need to test a deprecated or otherwise ill-advised practice. But this is not the situation here. Assuming all your data will arrive in a single `data` event is a gotcha of the language that we should do our level best to discourage.

But this is a largely philosophical position. If it lands as-is, then heck, I can just go open a `good first contribution` issue for changing it. :-D